pulp_certguard/app/models.py

```diff
@@ -119,6 +119,7 @@ def permit(self, request):
 
     @staticmethod
     def _create_rhsm_cert_from_pem(unquoted_certificate):
+        get_rhsm()
```

Comment:
This call is not necessary since the function earlier provides the check. I can remove myself in a followup PR to fix one or two things I saw before I release. Thanks!